release-notes/swan-lake-alpha4-release-note.md

```diff
@@ -92,6 +92,9 @@ To view bug fixes, see the [GitHub milestone for Swan Lake <VERSION>](https://gi
 
 #### Improvements
 
+- Introduced a single coverage-report.xml file per package instead of per module coverage-report.xml files. This coverage xml file can be found at 
```

Comment:
```suggestion
- Introduced a single `coverage-report.xml` file per package instead of per `module coverage-report.xml` files. This coverage XML file can be found at 
```